src/DependencyInjection/FeatureFinderFactoryPass.php

```diff
@@ -31,6 +32,11 @@ public function process(ContainerBuilder $container): void
             ->addArgument(new Reference(ToggleConfig::class))
             ->addArgument(new Reference(ChainToggleStrategyFactory::class));
 
+        Assert::keyExists($mergedConfig, 'driver');
+        Assert::string($mergedConfig['driver']);
+        Assert::keyExists($mergedConfig, 'driver_options');
```

Comment:
This field is mandatory? I'm not sure if we should only add if the driver is `chain`. What do you think?

Reply:
I thought that it must exist because if `$mergedConfig['driver`]` is different from "dbal" it will try to evaluate the `in_array` statement of the condition.